web/sites/default/config/default/webform.webform.edit_your_cssn_roles.yml

```diff
@@ -23,42 +23,13 @@ elements: |-
     '#type': checkboxes
     '#title': 'Roles:'
     '#options':
-      mentor: 'Mentor -- Provides or is willing to provide guidance, advice, feedback and support to a mentee. '
+      mentor: 'mentor -- Provides or is willing to provide guidance, advice, feedback and support to a mentee. '
       student: |
-        Student  -- A person enrolled in an institution of higher learning
-      research_computing_facilitator: 'Research Computing Facilitator -- Assists researchers with accessing data, study design, statistical and analytical methods, and coordinates the use of computing services, software licensing, and associated technical support for new and ongoing research projects and academic endeavors.'
-      research_software_engineer: 'Research Software Engineer -- Works with researchers to gain an understanding of the problems they face, and then develops, maintains and extends software to provide the answer'
-      ci_systems_engineer: 'CI Systems Engineer -- Provides direct centralized support on scientific computing to campus researchers to enhance computational thinking and modern computational methods'
-      researcher: 'Researcher / Educator -- A person doing research or teaching at an institution of higher learning'
-  academic_status_markup:
-    '#type': webform_markup
-    '#states':
-      visible:
-        ':input[name="roles_[student]"]':
-          checked: true
-    '#markup': '<h4>Academic Status</h4>'
-  academic_status_select2:
-    '#type': select
-    '#title': 'Academic Status'
-    '#title_display': none
-    '#options':
-      first: '1st year undergraduate'
-      second: ' 2nd year undergraduate'
-      third: ' 3rd year undergraduate'
-      fourth: ' 4th year undergraduate'
-      masters: ' Masters student'
-      phd: ' PhD student'
-      postdoc: 'Postdoctoral student'
-      fellow: Fellow
-      other: 'I am not in an academic program but interested in shifting focus to research computing facilitation'
-    '#select2': true
-    '#required': true
-    '#required_error': 'Please select an academic status'
-    '#states':
-      visible:
-        ':input[name="roles_[student]"]':
-          checked: true
-    '#states_clear': false
+        student-facilitator -- A person enrolled in an institution of higher learning
```

Comment:
Looks like the Academic Status field got removed. Did you pull --rebase to the latest version before creating this branch?